Source/Adobe.Target.Client/Util/TargetConstants.cs

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+namespace Adobe.Target.Client.Util
+{
+    /// <summary>
+    /// Target Constants
+    /// </summary>
```

Comment:
Could we collapse everything into a single class, I am talking about all the constants.

Reply:
Right, let me add the internal constants from Constants to same TargetConstants class